README.md

```diff
@@ -374,10 +377,23 @@ dablin_gtk -d ~/bin/dab2eti -c 11D -C 5C,7B,11A,11C,11D -s 0xd911
 ```
 
 You also can use an Airspy or Airspy mini, but you have to specify the
-gain in the `-C` parameter as the auto gain does not work right here.
+`-G` parameter.
 
 ```sh
-dablin_gtk -D eti-cmdline -d ~/bin/eti-cmdline-airspy -c 11D -C 11D:80
+dablin_gtk -D eti-cmdline -d ~/bin/eti-cmdline-airspy -c 11D -C 11D -G
```

Comment:
For consistency and easier reading, please adhere to the parameter order used in existing examples i.e. specify `-d` before `-D`, here and also below.

Reply:
Hmm, actually it makes more sense to specify `-D` before `-d`, as the former first selects the type of source, before the location of the binary is stated. So I instead aligned the other two existing examples accordingly.